### PR TITLE
Update install.rst

### PR DESCRIPTION
**Pull request by btozer**
**Description of proposed changes**

Include explicit instruction for how to set GMT_LIBRARY_PATH

 

Fixes #

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

### CODE REVIEW

File: doc/install.rst [hunk 1]
```diff
@@ -133,4 +133,7 @@ This can happen if you have multiple versions of GMT installed.
 
 You can tell PyGMT exactly where to look for ``libgmt`` by setting the
 ``GMT_LIBRARY_PATH`` environment variable.
-This should be set to the directory where ``libgmt.so`` (or ``.dylib``) is found.
+This should be set to the directory where ``libgmt.so`` (or ``.dylib``) is found 
+e.g. in a terminal run::
+   
+   export GMT_LIBRARY_PATH='"YOUR_PATH_TO_GMT6"/lib'
```
Comment (weiji14) [inline doc/install.rst]:
```suggestion
   export GMT_LIBRARY_PATH=$HOME/anaconda3/envs/pygmt/lib
```

File: doc/install.rst [hunk 2]
```diff
@@ -133,4 +133,7 @@ This can happen if you have multiple versions of GMT installed.
 
 You can tell PyGMT exactly where to look for ``libgmt`` by setting the
 ``GMT_LIBRARY_PATH`` environment variable.
-This should be set to the directory where ``libgmt.so`` (or ``.dylib``) is found.
+This should be set to the directory where ``libgmt.so`` (or ``.dylib``) is found
```
Comment (weiji14) [inline doc/install.rst]:
```suggestion
This should be set to the directory where ``libgmt.so``, ``libgmt.dylib`` or ``gmt.dll``
can be found for Linux, MacOS and Windows respectively.
```

**Comment by weiji14** [comment]
@seisman could you merge this in? It's saying I'm not authorized to merge into the master branch.

**Comment by seisman** [comment]
@btozer Thanks for your contributions!

@weiji14 Not sure why you can't merge it, since you're now already the team member. Possibly because the PR was opened before you joined the team?

**Comment by weiji14** [comment]
> @weiji14 Not sure why you can't merge it, since you're now already the team member. Possibly because the PR was opened before you joined the team?

Ah that's interesting, but I guess it kinda makes sense. Anyways, good work @btozer!

**Pull request merged**